Samples/Client/SwiftClient/SwiftClient/VideoStreamViewController.swift

```diff
@@ -169,18 +171,35 @@ class VideoStreamViewController: UIViewController {
         let attitude = deviceMotion.attitude
         prevYaw = yaw
         prevPitch = pitch
+        prevRoll = roll
         yaw = CGFloat(attitude.yaw)
         pitch = CGFloat(attitude.pitch)
+        roll = CGFloat(attitude.roll)
         let dheading = yaw - prevYaw
         let dpitch = pitch - prevPitch
+        let droll = roll - prevRoll
         // do not send movement that is too small
         if abs(dheading) < 0.005 && abs(dpitch) < 0.005 {
             return
         }
+
         prevNavHeading = navHeading
         prevNavPitch = navPitch
-        navHeading = prevNavHeading - dheading
-        navPitch = prevNavPitch + dpitch
+
+        if UIDevice.current.orientation == .landscapeLeft {
+            print("left")
```

Comment:
nit: can we remove these prints, seems like the rest of the nearby deviceMotion code doesn't log any info, and if we're confident this works i think we can avoid these statements 🥇